src/utilities/compareSCO.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash';
```

Comment:
we don't want all of lodash here, it's too expensive.
Can you modify this to use the single `isEqual`?
```
import isEqual from 'lodash/isEqual';
...
return isEqual(defaultState, SCOcomparator);
```